services/core-api/app/api/now_applications/response_models.py

```diff
@@ -3,23 +3,7 @@
 
 from app.api.parties.response_models import PARTY
 from app.api.mines.response_models import MINE_DOCUMENT_MODEL, PERMIT_AMENDMENT_SHORT_MODEL, MINE_TYPE_MODEL
-
-DOCUMENT_TEMPLATE_FIELD_MODE = api.model(
-    'DocumentTemplateFieldModel', {
-        "id": fields.String,
-        "label": fields.String,
-        "type": fields.String,
-        "placeholder": fields.String,
-        "required": fields.Boolean(default=False),
-        "context-value": fields.String,
-        "read-only": fields.Boolean(default=False),
-    })
-
-DOCUMENT_TEMPLATE_MODEL = api.model(
-    'DocumentTemplateModel', {
-        'document_template_code': fields.String,
-        'form_spec': fields.List(fields.Nested(DOCUMENT_TEMPLATE_FIELD_MODE, skip_none=True))
-    })
+from app.api.document_generation.response_models import DOCUMENT_TEMPLATE_MODEL
```

Comment:
does this actually need to be imported here?

Reply:
Yeah, it is used below.